lib/lambda/getPackageActions.test.ts

```diff
@@ -1,100 +1,77 @@
-import { describe, it, expect, vi, beforeEach } from "vitest";
 import { APIGatewayEvent } from "aws-lambda";
-import { handler } from "./getPackageActions";
-import { response } from "libs/handler-lib";
-import { getAvailableActions } from "shared-utils";
-import { getPackage } from "../libs/api/package/getPackage";
+import { getRequestContext } from "mocks";
 import {
-  getAuthDetails,
-  isAuthorizedToGetPackageActions,
-  lookupUserAttributes,
-} from "../libs/api/auth/user";
-
-vi.mock("libs/handler-lib", () => ({
-  response: vi.fn(),
-}));
-
-vi.mock("shared-utils", () => ({
-  getAvailableActions: vi.fn(),
-}));
-
-vi.mock("../libs/api/package/getPackage", () => ({
-  getPackage: vi.fn(),
-}));
-
-vi.mock("../libs/api/auth/user", () => ({
-  getAuthDetails: vi.fn(),
-  isAuthorizedToGetPackageActions: vi.fn(),
-  lookupUserAttributes: vi.fn(),
-}));
+  GET_ERROR_ITEM_ID,
+  HI_TEST_ITEM_ID,
+  NOT_FOUND_ITEM_ID,
+  WITHDRAWN_CHANGELOG_ITEM_ID,
+} from "mocks/data/items";
+import { describe, expect, it } from "vitest";
+import { handler } from "./getPackageActions";
 
 describe("getPackageActions Handler", () => {
-  beforeEach(() => {
-    vi.clearAllMocks();
-  });
-
   it("should return 400 if event body is missing", async () => {
     const event = {} as APIGatewayEvent;
 
-    await handler(event);
+    const res = await handler(event);
 
-    expect(response).toHaveBeenCalledWith({
-      statusCode: 400,
-      body: { message: "Event body required" },
-    });
+    expect(res).toBeTruthy();
+    expect(res.statusCode).toEqual(400);
   });
 
   it("should return 401 if not authorized to view resources from the state", async () => {
-    const packageData = { found: true, _source: { state: "test-state" } };
-    (getPackage as vi.Mock).mockResolvedValueOnce(packageData);
-    (isAuthorizedToGetPackageActions as vi.Mock).mockResolvedValueOnce(false);
+    const event = {
+      body: JSON.stringify({ id: HI_TEST_ITEM_ID }),
+      requestContext: getRequestContext(),
+    } as APIGatewayEvent;
 
+    const res = await handler(event);
+
+    expect(res).toBeTruthy();
```

Comment:
truthy is a just a quick check to see if it has a value. I don't strictly need to do it, but otherwise it complains aobut `res.statusCode` and `res.body` below it